src/test/regress/sql/failure_setup.sql

```diff
@@ -2,4 +2,4 @@ SELECT citus.mitmproxy('conn.allow()');
 
 -- add the workers
 SELECT master_add_node('localhost', :worker_1_port);  -- the second worker
-SELECT master_add_node('localhost', :worker_2_port + 2);  -- the first worker, behind a mitmproxy
+SELECT master_add_node('localhost', :worker_2_proxy_port);  -- the first worker, behind a mitmproxy
```

Comment:
Why does this say _the first worker_ when it's `worker2`?

Reply:
oops